tools/clang/test/taef_exec/lit.cfg

```diff
@@ -24,45 +25,64 @@ if te is None:
     if site_cfg and os.path.exists(site_cfg):
         lit_config.load_config(config, site_cfg)
         raise SystemExit
+bin_dir = os.path.join(config.llvm_obj_root, config.llvm_build_mode, 'bin')
 
 # TAEF only runs on Windows
 if platform.system() != 'Windows':
   config.unsupported = True
 else:
-  test_dll = os.path.join(config.llvm_obj_root, config.llvm_build_mode, 'bin', 'ExecHLSLTests.dll')
+  dxil_dll = os.path.join(bin_dir, 'dxil.dll')
+  if os.path.isfile(dxil_dll):
+    print(str.format("\nUsed {} as validator\n", dxil_dll))
+  else:
+    print(str.format("\nCannot find validator in {}\n", bin_dir))
+    dxexp = os.path.join(bin_dir, 'dxexp.exe')
+    result = subprocess.run([dxexp], capture_output=True)
+    no_experimental = "Experimental shader model feature failed"
+    out = lit.util.convert_string(result.stdout)
+    if no_experimental in out:
+        print("""Experimental shader model feature is not enabled.
+  Execution test require dxil.dll as validator to sign dxil container or enable experimental shader model feature to avoid signature check.
+              """)
+        config.unsupported = True
+
+skip_taef_exec = lit_config.params.get('skip_taef_exec', None)
```

Comment:
Why not move this up and the code in the prior else down to when we know exec tests are enabled?  If you're skipping execution tests, it would be odd to get errors about the execution tests still.

Reply:
Fixed.